app/component/IndexPage.js

```diff
@@ -148,8 +148,11 @@ class IndexPage extends React.Component {
       !this.props.origin.address &&
       this.props.locationState?.hasLocation &&
       this.props.locationState;
-
-    if (config.startSearchFromUserLocation && currentLocation) {
+    if (
+      config.startSearchFromUserLocation &&
+      currentLocation &&
```

Comment:
One minor thing more: no ned to check config.startSearchFromUserLocation again, it was alredy checked when setting currentLocation.